src/main/java/com/open/drive/user/User.java

```diff
@@ -0,0 +1,35 @@
+package com.open.drive.user;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document
+public class User {
+    @Id
+    private String id;
+    private String name;
+
+    public User() {
+    }
```

Comment:
Is this default constructor needed?
I don't see its call / maybe we can add some default values?

Reply:
Basically it is not needed. As per documentation is some JPA empty constructor is needed, but not in that case. It was created and not removed.